tests/performance/test_suite.py

```
@@ -0,0 +1,150 @@
+    def test_report_generation(self):
+        """Test performance report generation"""
+        # Add some data
+        self.manager.start_operation("test_op")
+        time.sleep(0.1)
+        self.manager.end_operation("test_op")
+        
+        report = self.manager.get_performance_report()
+        self.assertIn("monitoring", report)
+        self.assertIn("optimization", report)
+
+def run_tests():
```

Ruff (0.8.2)

Missing return type annotation for public function `run_tests`

Add return type annotation: `None`

(ANN201)